user_sync/connector/directory_ldap.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, caller_options):
         self.user_given_name_formatter = LDAPValueFormatter(options['user_given_name_format'])
         self.user_surname_formatter = LDAPValueFormatter(options['user_surname_format'])
         self.user_country_code_formatter = LDAPValueFormatter(options['user_country_code_format'])
+        self.user_memberof_format_formatter = LDAPValueFormatter(options['user_memberof_format'])
```

Comment:
Let's change the name of the config option so we don't group it with the other `user_` config options. Those options govern how user attributes are mapped. Those attributes translate directly to Admin Console user attributes. The group membership attribute is a special case. It also shouldn't be a "format" variable since it doesn't make sense to make it formattable. So we can remove this line to create a formatter.

I suggest naming it something like `group_member_attribute`.